### PR TITLE
Implementing RemoteWebDriver pipeline functionality

### PR DESCRIPTION
**Pull request by hankjtaylor**
Implementation for using RemoteWebDriver capabilities

### CODE REVIEW

File: groovy/JenkinsUtils.groovy [hunk 1]
```diff
@@ -9,6 +9,20 @@ void runTests(propertyFile) {
...
+  sh "cp devops/f4a/" + propertyFile + " f4a/FitNesseForAppian/fitnesse-automation.properties"
+  startWebDriverContainer()
+  dir("f4a/FitNesseForAppian") {
+    sh script: "bash ./runFitNesseTest.sh"
+  }
+}
+
+void startWebDriverContainer() {
+  sh "docker-compose -f docker/docker-compose.yml up firefox &"
+  sh "docker-compose -f docker/docker-compose.yml up chrome &"
```
Comment (cobylu) [inline groovy/JenkinsUtils.groovy]: not specifying the service would bring up both of them automatically. Do you think that specifying them individually  makes it significantly easier for them to modify for their own purposes?

Comment (hankjtaylor) [reply groovy/JenkinsUtils.groovy]: Nope, good catch

File: groovy/JenkinsUtils.groovy [hunk 2]
```diff
@@ -9,6 +9,15 @@ void runTests(propertyFile) {
   }
 }
 
+void runTestsDocker(propertyFile) {
+  sh "cp devops/f4a/" + propertyFile + " f4a/FitNesseForAppian/fitnesse-automation.properties"
+  sh "docker-compose -f docker/docker-compose.yml up &"
+  sleep(time:10, unit:"SECONDS")
```
Comment (samratjha96) [inline groovy/JenkinsUtils.groovy]: What is the purpose of this sleep?

Comment (hankjtaylor) [reply groovy/JenkinsUtils.groovy]: give the docker-compose time to come up. ideally, the better solution may be to pipe into a file and wait until we see that the "webdrive has started on port 4444", but this i think works until that solution is available

Comment (samratjha96) [reply groovy/JenkinsUtils.groovy]: This bash script will check for a container built with a selenium image to be up and running. I think this should eliminate the need to write to a file and tail the log. Unless there's a scenario where the container can be up and running without the server actually connecting to the port. 
```suggestion
  while [ -z "$(docker ps -a | awk '{print $1, $2}' | grep selenium |
          awk '{print $1}'  ]; do
    :
  done
```

**Comment by hankjtaylor** [comment]
@cobylu @samratjha96 testing passed -- re-up that CR and we'll merge upon approval!

**Comment by cobylu** [comment]
LGTM 🍣

**Comment by cobylu** [comment]
LGTM 🍣

**Comment by samratjha96** [comment]
LGTM 👨‍🎤

**Pull request merged**